schemas/rule.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: rule
+  title: Rule
+  group: 2
+  short: Fields to capture details about rules used by observers and agents to trigger events and/or actions.
+  description: >
+   Rule fields are used to capture the specifics of any observer or agent rules that are associated with a given action or event.
+   Typical sources of rule data might include any of the following: acl enabled network access equipment, network admission control 
+   platforms, network or host IDS/IPS, network firewalls, web access firewalls, url filters, etc.   
+  
+  type: group
+  fields:
+
+    - name: id
+      level: extended
+      type: keyword
+      short: Rule ID
+      description: >
+        A rule ID that is unique within the scope of an agent, observer, or other entity using the rule for detection of this event
+
+      example: 101 
+
+    - name: uuid
+      level: extended
+      type: keyword
+      short: Rule UUID
+      description: >
+        A rule ID that is unique within the scope of a set or group of agents, observers, or other entities using the rule for detection of this event.
+
+      example: 1100110011
+
+    - name: version
+      level: extended
+      type: keyword
+      short: Rule Version
+      description: The version / revision of the rule being used for analysis.
+      example: 1.1
+
+    - name: name
+      level: extended
+      type: keyword
+      short: Rule Name
+      description: The name of the rule generating the event.
+      example: Filter_DOH
+
+
+    - name: description
+      level: extended
+      type: keyword
+      short: Rule Description
+      description: The description of the rule generating the event.
+      example: Block requests to public DNS over HTTPS / TLS sites
+
+    - name: type
+      level: extended
+      type: keyword
+      short: Rule Type
+      description: >
+         A categorization value keyword used by the entity using the rule for detection of this event
+
+      example: DNS
+      
+    - name: ruleset
+      level: extended
+      type: keyword
+      short: Rule Ruleset
+      description: >
+         The parent category of the rule used to create this event.
```

Comment:
can we change "The parent category of the rule used to create this event." to "name of the ruleset, policy, group, or parent category in which the rule used to generate this event is a member."

Reply:
done